pkg/scheduler/eventhandlers.go

```diff
@@ -362,8 +362,7 @@ func (sched *Scheduler) skipPodUpdate(pod *v1.Pod) bool {
 func addAllEventHandlers(
 	sched *Scheduler,
 	informerFactory informers.SharedInformerFactory,
-	podInformer coreinformers.PodInformer,
-) {
+	podInformer coreinformers.PodInformer) {
```

Comment:
either one should be good.

Reply:
yeah, but I think it's easy to make a mistake :)

Reply:
I don't think the proposed change is always better than the current style. For instance, with the current style, you can delete a parameter by simply deleting that whole line; while with the proposed change, if you want to delete the last parameter, you have to manually move `){` one line above.